### ISSUE TITLE
Add stroke for fonts

### ISSUE THREAD

**Issue by danknil** [8bitPit/Niagara-Issues#2737]
### Is your feature request related to a problem?

I think there must be a way to make a stroke for font for better visibility. For example here is my setup with wallpapers, that doesnt work well with white or black font colors, but icons that have stroke around them are clearly visible. 

![Screenshot_20240607-123311_Niagara Launcher](https://github.com/8bitPit/Niagara-Issues/assets/65212334/5c109399-a3fc-403e-bb99-8b0d4ce7043d)

![Screenshot_20240607-123259_Niagara Launcher](https://github.com/8bitPit/Niagara-Issues/assets/65212334/a4f3001f-3df1-469c-a67b-3b94dc88b0ef)

### Describe the solution you'd like

Add an option to enable font stroke with opposite color(e.g. if font color is white, then make black stroke and vise versa)

### Feature Request Procedure

- [X] I have read the [knowledge base article on feature requests](https://help.niagaralauncher.app/article/44-feature-requests) and followed the steps mentioned

**Comment by contact-maximux** [comment]
Yes absolutely required. There should be option for font-shadow so that on any kind of wallpaper it is visible

**Comment by NiagaraSupport** [comment]
Hey there, thanks for the suggestion.
It's unlikely that we add an option to increase the text-shadow, but additional fonts for the launcher should be coming soon. 
I'll make sure to look for styles(like shaded fonts) that improve visibility on wallpaper.
